data/tests/tests_common.txt

```diff
@@ -205,3 +583,21 @@ test "Depart"
 		input
 		branch zoom_restore_loop
 			"test: Depart: zoom_restore_counter" > 0
+
+
+test "Sell first ship"
+	status partial
+	description "Test procedure that handles selling of the first ship in the list."
+	sequence
+		# Goto shipyard
+		input
+			key "s"
+		# Sell the first ship
+		input
+			key "s"
+		input
+			key "Return"
+		# Go back to main menu
```

Comment:
Technically not the main menu, but the planet menu or whatever it is called.

Reply:
Good point, I'll use planet menu.